### ISSUE TITLE
im.MatrixColor documentation

### ISSUE THREAD

**Issue by trajano** [renpy/renpy#1931]
In https://www.renpy.org/doc/html/displayables.html?highlight=brightness#im-matrixcolor instead of *image manipulator* perhaps just say image file name just to clarify

This does not work
```
image button back idle = "screens/ui/twotone_skip_previous_white_18dp.png"
image button back insensitive = im.MatrixColor(
    "button back idle",
    im.matrix.brightness(-0.5))
```

but this does

```
image button back idle = "screens/ui/twotone_skip_previous_white_18dp.png"
image button back insensitive = im.MatrixColor(
    "screens/ui/twotone_skip_previous_white_18dp.png",
    im.matrix.brightness(-0.5))
```

**Comment by renpytom** [comment]
Except that's not right. Since this perfectly happy to take other image manipulators, like im.Data or im.Crop.

**Comment by trajano** [comment]
Should `"button back idle"` be considered an image manipulator in my example above?  If not maybe have a link to what an *Image manipulator* is.

**Comment by renpytom** [comment]
No. You might want to look at the documentation for image manipulators: 

https://www.renpy.org/doc/html/displayables.html#image-manipulators

**Comment by trajano** [comment]
Oh finally I just noticed the example of how to use the `Transform()` function right smack at the bottom of that page https://www.renpy.org/doc/html/displayables.html#displayable-prefixes

Given that I may be able to do what I intended to to.

Closing this off, but something to consider having in **bold** letters saying that image manipulators should be avoided or perhaps taken out of the page into a separate page so noobs (e.g. me) don't try to use it accidentally.
